apps/mail/app/(routes)/settings/shortcuts/page.tsx

```
@@ -35,20 +72,57 @@ export default function ShortcutsPage() {
   );
 }
 
-function Shortcut({ children, keys }: { children: ReactNode; keys: string[] }) {
+function Shortcut({ children, keys, action }: { children: ReactNode; keys: string[]; action: string }) {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'Shortcut'. Consider to delete it or rename it.

'Shortcut' is defined here:


(lint/suspicious/noRedeclare)